soh/soh/SohGui/SohMenuEnhancements.cpp

```diff
@@ -560,6 +560,10 @@ void SohMenu::AddMenuEnhancements() {
         .RaceDisable(false)
         .Options(IntSliderOptions().Min(4).Max(6).DefaultValue(6).Tooltip(
             "Space between text characters (useful for HD font textures)."));
+    AddWidget(path, "Use 3D Pre-Rendered Scenes", WIDGET_CVAR_CHECKBOX)
+        .CVar(CVAR_SETTING("3DSceneRender"))
+        .RaceDisable(false)
+        .Options(CheckboxOptions().Tooltip("Replaces 2D with 3D Pre-rendered backgrounds in Scenes."));
```

Comment:
```suggestion
    AddWidget(path, "Disable 2D Pre-Rendered Scenes", WIDGET_CVAR_CHECKBOX)
        .CVar(CVAR_ENHANCEMENT("3DSceneRender"))
        .RaceDisable(false)
        .Options(CheckboxOptions().Tooltip("Disables 2D pre-rendered backgrounds. Enable this when using a mod that implements 3D backdrops for these areas."));
```

Previous description isn't really true, and will likely lead to people enabling this without having a mod to properly step in to make use of this.